libs/providers/flagd-web/project.json

```
@@ -58,23 +58,11 @@
         }
       ]
     },
-    "pullTestHarness": {
-      "executor": "nx:run-commands",
-      "options": {
-        "commands": [
-          "git submodule update --init spec",
-          "rm -f -r ./src/e2e/features/*",
-          "cp -v ./spec/specification/assets/gherkin/evaluation.feature ./src/e2e/features/"
-        ],
-        "cwd": "libs/providers/flagd-web",
-        "parallel": false
-      }
-    },
     "e2e": {
       "executor": "nx:run-commands",
       "options": {
         "commands": [
-          "npx jest"
+          "npx jest --detectOpenHandles --runInBand"
```

```suggestion
          "npx jest
```

Yep, works fine without this.